src/FittedImage.js

```diff
@@ -103,12 +107,20 @@ const FittedImage = React.createClass({
     this.setState({
       status: states.LOADED
     });
+
+    if (background || !modern) {
```

Comment:
My reasoning here is that the event handlers are already spread onto the actual image element and will be called by the browser. The only case we need to manually call them is when we are using the fallback. The alternative to this is not applying `onLoad` and `onError` to the actual image element. Let me know which method you like most and I'll make the changes!